sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/CloudEventRequestContent.cs

```diff
@@ -54,25 +56,28 @@ private void EnsureSerialized()
                 return;
             }
 
-            string currentActivityId = null;
-            string traceState = null;
-            Activity currentActivity = Activity.Current;
-            if (currentActivity != null && currentActivity.IsW3CFormat())
+            if (_isDistributedTracingEnabled)
```

Comment:
I'm wondering if we will need to document this as a breaking change?

Reply:
/cc @KrzysztofCwalina @tg-msft

Reply:
Feel free to merge this.. This is just more of a question as to how we present the change in change logs.